app/Http/Controllers/TaskSchedulerController.php

```diff
@@ -49,7 +54,12 @@ class TaskSchedulerController extends BaseController
 
     public function index()
     {
-        return Scheduler::where('company_id', auth()->user()->company()->id)->cursor();
```

Comment:
for this we just pass the builder into fractal, so ->cursor() isn't required at this point.